torch/distributed/rpc/api.py

```diff
@@ -129,6 +131,16 @@ def get_worker_info(worker_name=None):
     else:
         return _agent.get_worker_info()
 
+@_require_initialized
+def get_rpc_timeout():
+    """
+    Retrieve the timeout for all RPCs that was set during RPC initialization.
+
+    Returns:
+        `datetime.timedelta` instance indicating the RPC timeout.
+    """
+    return _agent._get_rpc_timeout()
```

Comment:
Open question about whether we should expose this to the user or not, but I think since it is configurable by the user we should have a getter for it as well.